### PR TITLE
Refactoring and simplifying of variables related to power plant specifications

### PR DESCRIPTION
**Pull request by danielhuppmann**
This PR implements the refactoring of the fuel-type list (implemented for electricity at the secondary-energy level, see #64) for the variables representing characteristics of power plants.

A few clarifications are needed to finalize this:

 - Are we ok that there are multiple variables for operational cost
   - `Operational Cost|Constant Term|...`
   - `Operational Cost|Linear Term|...`
   - `Operation and Maintenance Cost|...`
  What are the differences?
 - Are we fine with the variables `Rate Frequency Containment Reserve` and `Rate Automatic Frequency Containment Reserve`, or should we drop the `Rate`?
 - Profile is defined in energy terms for hydro-reservoir inflow but as percentage (of what?) for other renewables... We should harmonize this before starting to use this.
 - What is meant with "Energy Storage System" - battery? compressed air?

### CODE REVIEW

File: nomenclature/definitions/variable/technology/power-plant.yaml [hunk 1]
```diff
@@ -3,638 +3,153 @@
...
-   description: Percentage of the Maximum Active Power that can be devoted to Frequency Containment Reserve
-                for a typical "simple Hydro reservoir " unit
-   unit: "%"
-
-Rate Automatic Frequency Restoration Reserve|Electricity|Hydro|Reservoir:
-   description: Percentage of the Maximum Active Power that can be devoted to Automatic Frequency Restoration Reserve
-                for a typical "simple Hydro reservoir " unit
+# specific variables for run-of-river hydro power plants
+Load Factor|Electricity|Hydro|Run of River|Profile:
+   description: Load of hourly loadfactors for "Run of River"
```
Comment (tburandt) [inline nomenclature/definitions/variable/technology/power-plant.yaml:152]: As the name of the variable was changed, I would also change the description
```suggestion
   description: Load of hourly load factors for "Run of River"
```

File: nomenclature/definitions/variable/technology/power-plant.yaml [hunk 2]
```diff
@@ -675,94 +190,16 @@ Stored Energy Inventory|Electricity|Energy Storage System:
...
-   description: Number of units of kind Photovoltaic power plant
-   unit:
-
-Maximum Active power|Electricity|Solar:
-   description: Maximum active power of one Photovoltaic power plant
-   unit: MW
-
-LoadFactor|Electricity|Solar|Profile:
+Load Factor|Electricity|Solar|Profile:
    description: Timeserie of hourly loadfactors for Photovoltaic power plants
```
Comment (tburandt) [inline nomenclature/definitions/variable/technology/power-plant.yaml:194]:
```suggestion
   description: Timeseries of hourly load factors for Photovoltaic power plants
```

File: nomenclature/definitions/variable/technology/power-plant.yaml [hunk 3]
```diff
@@ -675,94 +190,16 @@ Stored Energy Inventory|Electricity|Energy Storage System:
...
-   description: Number of units of kind Wind Onshore power plant
-   unit:
-
-Maximum Active power|Electricity|Wind|OnShore:
-   description: Maximum active power of one Wind Onshore power plant
-   unit: MW
-
-LoadFactor|Electricity|Wind|OnShore|Profile:
+Load Factor|Electricity|Wind|Onshore|Profile:
    description: Timeserie of hourly loadfactors for Onshore Windpower
```
Comment (tburandt) [inline nomenclature/definitions/variable/technology/power-plant.yaml:199]:
```suggestion
   description: Timeseries of hourly load factors for Wind Onshore power plants
```

File: nomenclature/definitions/variable/technology/power-plant.yaml [hunk 4]
```diff
@@ -675,94 +190,16 @@ Stored Energy Inventory|Electricity|Energy Storage System:
...
-   description: Number of units of kind Wind OffShore power plant
-   unit:
-
-Maximum Active power|Electricity|Wind|OffShore:
-   description: Maximum active power of one Wind OffShore power plant
-   unit: MW
-
-LoadFactor|Electricity|Wind|OffShore|Profile:
+Load Factor|Electricity|Wind|OffShore|Profile:
    description: Timeserie of hourly loadfactors for Offshore Windpower
```
Comment (tburandt) [inline nomenclature/definitions/variable/technology/power-plant.yaml:204]:
```suggestion
   description: Timeseries of hourly load factors for Wind Offshore power plants
```

**Comment by danielhuppmann** [comment]
thanks @tburandt, I accepted the suggestions (but they will probably change again depending on the conclusion about load factor convention)

**Comment by arght** [comment]
- Are we fine with the variables Rate Frequency Containment Reserve and Rate Automatic Frequency Containment Reserve, or should we drop the Rate?

Usually the values that are included here depend on the demand, variable RES and largest generating unit. From this point of view I would rather prefer to specify these values in MW, not as a rate, which I assume would be in p.u. or %

**Comment by arght** [comment]
•	Profile is defined in energy terms for hydro-reservoir inflow but as percentage (of what?) for other renewables... We should harmonize this before starting to use this.

Natural hydro inflows are usually expressed in m3/s or GWh for each reservoir. I understand that profile for variable RES can be expressed as p.u. of their installed capacity. Alternatively I suggest to express it directly as MW.

**Comment by arght** [comment]
•	What is meant with "Energy Storage System" - battery? compressed air?
ESS is any system able to store and generate energy (pumped-storage hydro, battery, compressed air energy system, etc.)

**Comment by sandrinecharousset** [comment]
Regarding the variables Rate Frequency Containment Reserve and Rate Automatic Frequency Containment Reserve, it is important to keep sth like rate or another word meaning it is a percentage OR these values could be expressed as maximum Frequency Containment Reserve (which is equAl to maximum active power * rate frequency....)
I am indifferent to this

**Comment by sandrinecharousset** [comment]
All profiles for res could be expressed in MW or % of a maximal active power, as you prefer
This mean we would have a fixed max power AND a max power profile (keeping the |profile as for the inflows would be enough to distinguish)
Regarding inflows they should be in MWh not MW (equivalent if hourly but usually available data are daily or per 6h... but never saw hourly data...)

**Review by sandrinecharousset — APPROVED** [review]

**Comment by danielhuppmann** [comment]
closing in favor of #90 #93 #96

**Pull request closed without merging**